app/navigation/Navigator.scala

```diff
@@ -95,11 +95,28 @@ class Navigator @Inject()(appConfig: FrontendAppConfig)
     case EmployeeStartDatePage => employeeStartDateRoutes
     case PayPeriodQuestionPage =>
       payPeriodQuestionRoutes
+    case PartTimeQuestionPage => partTimeQuestionRoute
     case _ =>
       _ =>
         routes.RootPageController.onPageLoad()
   }
 
+  private def regularPayAmountRoute: UserAnswers => Call = { userAnswer =>
+    if (isPhaseTwo(userAnswer)) routes.PartTimeQuestionController.onPageLoad()
+    else routes.TopUpStatusController.onPageLoad()
+  }
+
+  private def partTimeQuestionRoute: UserAnswers => Call =
+    userAnswer =>
+      userAnswer
+        .getV(PartTimeQuestionPage)
+        .map {
+          case PartTimeYes if userAnswer.getList(PayDatePage).size > 1 => routes.PartTimePeriodsController.onPageLoad()
```

Comment:
I think this check is unnecessary as pay dates will always be > 1 as you are prompted to enter a date before furlough/claim start and then at least one more date after furlough/claim start but there is no harm in checking here I guess

Reply:
from the flow diagram if PartTimeYes if userAnswer.getList(PayDatePage).size < 2 => we go to /part-time-hours (this bit is not implemented yet) but i think we do need the check